lib/Paymill/Services/ResponseHandler.php

```diff
@@ -396,25 +479,24 @@ public function convertErrorToModel($response, $resourceName = null)
     }
 
     /**
-     * Validates the data responsed by the API
+     * Validates the data responded by the API
+     * Just checks the header status is successful.
      *
-     * Only Refund, Transaction and Preauthorization return an response_code
      * @param array $response
-     * @return boolean
+     *
+     * @return boolean True if valid
      */
     public function validateResponse($response)
     {
         $returnValue = false;
-        if ($response['header']['status'] == 200) {
```

Comment:
Because of the handling implemented on the merchants side. A response != 20000 was nerver valid until now. Valid could be implement as success, which is not the case any more.